src/components/FilmPoster/FilmPosterForm.tsx

```diff
@@ -44,10 +40,8 @@ const FilmPosterDisplay = ({ filmEPK, setIsEditing, isEditing }: any) => {
           type="file" 
           accept="image/*" 
         />
+        {isLoading && <p>Saving image...</p>}
         {reminder && <p>Must choose a file to save</p>}
```

Comment:
Nice additions, Rachel. This definitely helps to curate a better user experience.